README.md

```diff
@@ -50,11 +50,11 @@ pip install .
 
 This repo serves the purpose of fast prototyping [GFlowNet](https://arxiv.org/abs/2111.09266) (GFN) related algorithms. It decouples the environment definition, the sampling process, and the parametrization of the function approximators used to calculate the GFN loss. It aims to accompany researchers and engineers in learning about GFlowNets, and in developing new algorithms.
 
-Currently, the library is shipped with three environments: two discrete environments (Discrete Energy Based Model and Hyper Grid) and a continuous box environment. The library is designed to allow users to define their own environments. See [here](https://github.com/saleml/torchgfn/tree/master/tutorials/ENV.md) for more details.
+Currently, the library is shipped with three environments: two discrete environments (Discrete Energy Based Model and Hyper Grid) and a continuous box environment. The library is designed to allow users to define their own environments. See [here](./tutorials/ENV.md) for more details.
```

Comment:
NB: I have updated these links to be relative links instead of hard links to the master branch on GitHub. This has two advantages:

- If on a fork/branch, the links will correctly point to the version of the document on that fork/branch
- If you click the link in a local editor, it will navigate to the local file in the editor instead of via the browser

I think this is somewhat more optimal but I'm happy to revert these changes if it's not prefered!

Reply:
I'm thinking that this might break the docs on https://torchgfn.readthedocs.io/en/latest/ though... Maybe it's worth backing this out and then trying to address this more structurally in #123?